### PR TITLE
Text Component changes has been committed

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Text Custom"
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml]: @DeepaliAgrwl : keep Name as Text only

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                                    </column>
+                                </items>
+                            </columns>
+                        </items>
+                    </properties>
+                </items>
+            </tabs>
+        </items>
+    </content>
+</jcr:root>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml:182]: @DeepaliAgrwl : add source edit functionality too . 
image

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+                                                                        items="[lists#unordered,lists#ordered,lists#outdent,lists#indent]"
+                                                                        ref="lists"/>
+                                                                <paraformat
+                                                                        jcr:primaryType="nt:unstructured"
+                                                                        items="paraformat:getFormats:paraformat-pulldown"
+                                                                        ref="paraformat"/>
+                                                            </popovers>
+                                                        </inline>
+                                                        <dialogFullScreen
+                                                                jcr:primaryType="nt:unstructured"
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml]: @DeepaliAgrwl : provide table oftion in fullscreen. 
image

**Review by rishabh-behl — COMMENTED** [review]
@DeepaliAgrwl : pl incorporate comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml]: Change done

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml (hunk 2 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml:182]: @rishabh-behl  added this functionality in both inplace and full screen editing mode.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml (hunk 3 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml]: Added table option in full screen mode

File: ui.apps/src/main/content/META-INF/vault/filter.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-    <filter root="/apps/hclecomm"/>
+    <filter root="/apps/hclecomm" mode="merge/>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/META-INF/vault/filter.xml]: @DeepaliAgrwl : build is failing bacuse of this . Please correct with "

**Review by rishabh-behl — APPROVED** [review]

**Comment by rishabh-behl** [comment]
merging

**Pull request merged**